public/app/features/panel/metrics_panel_ctrl.ts

```diff
@@ -19,12 +19,12 @@ import { PanelModel } from 'app/features/dashboard/state';
 import { PanelQueryRunner } from '../query/state/PanelQueryRunner';
 
 class MetricsPanelCtrl extends PanelCtrl {
-  datasource: DataSourceApi;
+  datasource?: DataSourceApi;
```

Comment:
I remember we discussed something about babel, Angular and marking props as undefined. But I don't recall right now what it was, I think props will not exist or something like that.